docs/guides/included-middleware.md

```diff
@@ -0,0 +1,41 @@
+Crow contains some middlewares that are ready to be used in your application.
+
+Make sure you understand how to enable and use [middleware](../middleware/).
+
+## Cookies
+Include: `crow/middlewares/cookie_parser.h` <br>
+Examples: `examples/middlewars/example_cookies.cpp`
+
+This middleware allows to read and write cookies by using `CookieParser`. Once enabled, it parses all incoming cookies.
+
+Cookies can be read and written with the middleware context. All cookie attributes can be changed as well.
+
+```c++
+auto& ctx = app.get_context<crow::CookieParser>(request);
+std::string value = ctx.get_cookie("key");
+ctx.set_cookie("key", "value")
+    .path("/")
+    .max_age(120);
+```
+
+**Note**: Make sure the CookieParser is listed before other middleware that relies on it
```

Comment:
This should be changed to:
```
!!! note
    Make sure `CookieParser` is listed before any other middleware that relies on it.
```
That way the note will be formatted as a note with a border, rather than just being text.

Reply:
![image](https://user-images.githubusercontent.com/60552923/167154790-52ed42a0-0b5d-429a-8ca4-1a1399e8f5d0.png)
This is what a bordered note looks like (just for reference)